### DISCUSSION TITLE
Why is there no Get/PutObjectCallable in the S3CrtClient?

### DISCUSSION THREAD

**Discussion by enriquemartinezdev** [aws/aws-sdk-cpp#2305] [General]
The previous S3Client had both GetObjectCallable and PutObjectCallable methods, as well as their asynchronous and synchronous counterparts. However, S3CrtClient has synchronous, asynchronous and callable methods for all operations, except these. Is there a reason for it?

**Comment by jmklix** [comment]
To be honest I'm not sure why that design choice was made. It this something that you are looking to use? You can always create a feature request [here](https://github.com/aws/aws-sdk-cpp/issues/new?assignees=&labels=feature-request%2Cneeds-triage&template=feature-request.yml&title=%28short+issue+description%29) if it is something that you would like to be added to this sdk.
